addons/gecs_network/docs/sync-patterns.md

```
@@ -0,0 +1,212 @@
+        CN_NetworkIdentity.new(0),  # 0 = server-owned
+        C_Projectile.new(),
+        C_Velocity.new(),
+        C_Transform.new(),
+        C_DeathTimer.new(3.0),
+    ]
+    # NO CN_SyncEntity - this is what makes it spawn-only!
+```
+
+### Complete Spawn-Only Flow
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)